modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -91,7 +91,7 @@ metadata:
   namespace: default
 spec:
   scaleTargetRef:
-    apiVersion: v1 <3>
+    apiVersion: apps/v1 <3>
     kind: ReplicaSet <4>
```

Comment:
While this also uses `kind: Deployment` in 4.8+, `ReplicaSet` is also `apps/v1` so this backport is correct.